### PR TITLE
PICMI: warpx_file_min_digits

### PR DESCRIPTION
**Pull request by ax3l**
Add for all diagnostics the equivalent of `<diag>.file_min_digits`.

### CODE REVIEW

File: Python/pywarpx/picmi.py [hunk 1]
```diff
@@ -1218,6 +1221,7 @@ def initialize_inputs(self):
         self.diagnostic.intervals = self.period
         self.diagnostic.diag_type = 'Full'
         self.diagnostic.format = 'checkpoint'
+        self.diagnostic.file_min_digits = file_min_digits
```
Comment (ax3l) [inline Python/pywarpx/picmi.py]: Fails with:
```
NameError: name 'file_min_digits' is not defined
```

I don't understand why, help wanted @dpgrote :)

Comment (dpgrote) [reply Python/pywarpx/picmi.py]: It needs to be `self.file_min_digits`.

Comment (ax3l) [reply Python/pywarpx/picmi.py]: dang, of course. thanks :)

**Comment by ax3l** [comment]
@dpgrote now it works, ready for approval :)

**Review by dpgrote — APPROVED** [review]
Looks good!

**Pull request merged**